### PR TITLE
fix: match guests list API + built .apk file

### PR DESCRIPTION
**Pull request by SpQuyt**

### CODE REVIEW

File: src/components/Home/EventDetail/Guest/Unaccepted/index.js [hunk 1]
```diff
@@ -28,90 +42,115 @@ class Unaccepted extends Component {
...
   }
 
-  static navigationOptions = {
-    tabBarLabel: 'Chưa được chấp nhận',
+  generateNumbersList = (length) => {
+    const numbersList = [];
+    for (let i = 1; i <= length; i++) {
+      numbersList.push(i);
+    }
+    return numbersList;
```
Comment (namdaoduy) [inline src/components/Home/EventDetail/Guest/Unaccepted/index.js]: Move to utils

File: src/configs/index.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ import prodConfig from './prod';
 
 const env = REACT_APP_ENV;
 let envConfig = {};
+console.log(REACT_APP_ENV);
```
Comment (namdaoduy) [inline src/configs/index.js]: Remove

File: src/components/Home/EventDetail/Guest/Unaccepted/index.js (hunk 1 above)
Comment (SpQuyt) [reply src/components/Home/EventDetail/Guest/Unaccepted/index.js]: ok

File: src/configs/index.js (hunk 2 above)
Comment (SpQuyt) [reply src/configs/index.js]: ok

**Pull request merged**